conf/development.ini

```diff
@@ -90,6 +90,10 @@ py-tracebacker = /tmp/via-traceback-
 env = VIA_H_EMBED_URL=http://localhost:3001/hypothesis
 env = VIA_IGNORE_PREFIXES=http://localhost:5000/,http://localhost:3001/,https://localhost:5000/,https://localhost:3001/
 env = VIA_DEBUG=1
+env = VIA_BLOCKLIST_FILE=../conf/blocklist-dev.txt
+
+
+py-autoreload = true
```

Comment:
I don't think this actually works to do auto reloading in dev... but I want to leave it here as a niggle to remind me